integration/helpers.go

```diff
@@ -124,6 +124,29 @@ func AssertEqualError(t testing.TB, expected mongo.CommandError, actual error) b
 	return assert.Equal(t, expected, a)
 }
 
+// AssertEqualErrorMessage asserts that expected error is the same as actual, ignoring the Raw part.
+func AssertEqualErrorMessage(t testing.TB, expected mongo.CommandError, altMessage string, actual error) bool {
+	t.Helper()
+
+	a, ok := actual.(mongo.CommandError)
+	if !ok {
+		return assert.Equal(t, expected, actual)
+	}
+
+	// raw part might be useful if assertion fails
```

Comment:
If the assertion fails, it's unreachable, isn't it? there is a return on 133 line.Or which one assertion do you mean?
Raw is not used below.

Reply:
If it's not used now, maybe could this code be removed?

Reply:
Why it's not used?

Reply:
I just don't see where it's used.. In the error message when it prints when it fails?

Reply:
Yeah. When an assertion fails there would be a test output where the Raw part would be shown.